tvb_documentation/tutorials/tutorial_0_GettingStarted.rst

```diff
@@ -4,189 +4,220 @@
-can always work with the default account *admin*. Make sure you have
-enough disk space, all the projects add up to 2 GB.
+In this exercise we will assume that multiple users are working with the
+same TVB installation.
 
-#. Go to *Projects > List of all projects*.
 
-#. On the right column, bottom corner, click on Import project structure. 
+1. By default your are the *Admin* user.
 
-.. figure:: figures/ImportingProjects_Import.png
-   :alt: Click on Import project structure
+2. Change the admin email.
+
+.. figure:: figures/Users_ChangeAdminEmail.png
    :scale: 30%
 
-3. Select one of the projects. Click on *Upload*.
 
-.. figure:: figures/ImportingProjects_ImportOverlay.png
-   :alt: Click on Select files
-   :scale: 30%
+3. Create a user for someone else by registering a new user.
 
-.. figure:: figures/ImportingProjects_Upload.png
-   :alt: Upload.
+.. figure:: figures/Users_AddNewUser.png
    :scale: 30%
 
-4. Be patient.
+4. You’ll receive a notification email.
+
+5. Make sure the new user has been validated.
 
-.. figure:: figures/ImportingProjects_Wait.png
-   :alt: Be patient, it will take a few minutes.
+.. figure:: figures/Users_ValidateNewUser.png
+   :scale: 40%
+
+6. Login with the Admin account.
+
+--------------------------------------------
+
+Creating, Importing, Exporting and Deleting Projects
+---------------------------------------------------------
+
+We assume that you have already created an account on your machine. If not, you
+can always work with the default account *admin*.
+
+Normally when you start TVB you should have the *Default Project* from *tvb_data* already imported.
+If you don't have it, go to step 4 to see how you can import a new project.
+
+1. Go to *Project > List of all projects*.
+
+2. On the right column, bottom corner, click on *Create new project*.
+
+.. figure:: figures/Projects_ListOfAllProjects1.png
+   :alt: Click on Save Changes
    :scale: 30%
 
-5. You can now see the project on the *List of all projects*.
+3. Choose a project name (e.g. *NewProject*) and optionally add a description as well.
+   On the right column, bottom corner, click on *Save Changes* when you are done.
 
-.. figure:: figures/ImportingProjects_Done.png
-   :alt: The imported project is on the .
+.. figure:: figures/Projects_CreatingProjects.png
+   :alt: Click on Save Changes
    :scale: 30%
 
+   The new project will appear in the list of all projects.
 
-------------------------
+.. figure:: figures/Projects_ListOfAllProjects2.png
+   :alt: See the newly created project
+   :scale: 30%
 
-Adding A New User
----------------------------
+4. If you wish to import a project, on the right column, bottom corner,
+   click on *Import project structure*.
 
-In this exercise we will assume that multiple users are working with the
-same TVB installation.
+.. figure:: figures/Projects_Import.png
+   :alt: Click on Import project structure
+   :scale: 30%
 
+Click on *Upload*.
```

Comment:
would be more clear: Choose the ModelingEpilepsy.zip project and click on Upload?